<launch>
  <arg name="device_name"          default="/dev/ttyUSB0"/>
  <arg name="baud_rate"            default="1000000"/>
  <arg name="protocol_version"     default="2.0"/>

  <param name="device_name"        value="$(arg device_name)"/>
  <param name="baud_rate"          value="$(arg baud_rate)"/>
  <param name="protocol_version"   value="$(arg protocol_version)"/>

  <node name="open_manipulator_dynamixel_controller" pkg="open_manipulator_dynamixel_ctrl" type="dynamixel_controller" required="true" output="screen"/>
</launch>
